static/app/components/replays/replayView.tsx

```diff
@@ -10,50 +8,19 @@ import ReplayCurrentUrl from 'sentry/components/replays/replayCurrentUrl';
 import ReplayPlayer from 'sentry/components/replays/replayPlayer';
 import space from 'sentry/styles/space';
 
-// How much to reveal under the player, so people can see the 'pagefold' and
-// know that they can scroll the page.
-const BOTTOM_REVEAL_PIXELS = 70;
-
-const SCREEN_HEIGHT_DIVISOR = 2;
-
 type Props = {
   isFullscreen: boolean;
   toggleFullscreen: () => void;
 };
 
 function ReplayView({isFullscreen, toggleFullscreen}: Props) {
-  const containerRef = useRef<HTMLDivElement>(null);
-  const playerRef = useRef<HTMLDivElement>(null);
-
-  const [windowInnerHeight, setWindowInnerHeight] = useState(window.innerHeight);
-  const [playerHeight, setPlayerHeight] = useState(0);
-
-  useEffect(() => {
-    const onResize = debounce(() => {
-      setWindowInnerHeight(window.innerHeight);
-    });
-    window.addEventListener('resize', onResize);
-    return () => {
-      window.removeEventListener('resize', onResize);
-    };
-  }, []);
-
-  useEffect(() => {
-    const containerBottom =
-      (containerRef.current?.offsetTop || 0) + (containerRef.current?.offsetHeight || 0);
-    const playerOffsetHeight = playerRef.current?.offsetHeight || 0;
-    const calc =
-      windowInnerHeight - (containerBottom - playerOffsetHeight) - BOTTOM_REVEAL_PIXELS;
-    setPlayerHeight(Math.max(200, calc / SCREEN_HEIGHT_DIVISOR));
-  }, [windowInnerHeight]);
```

Comment:
This stuff was basically doing `height: 60vh;` on the element that is the container for the Player.

What's happening now on the page (with the v2 layout) is that we've got `<ResizePanel>` and setting the dimensions, so none of this math is needed.